src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.unit.spec.jsx

```diff
@@ -251,10 +251,6 @@ describe('ApplyForBenefits component', () => {
             );
           }),
         ).to.be.true;
-        // make sure the loading spinner is shown
```

Comment:
The only reason this spinner was visible before was because of the issue I fixed above that resulted in `enrollmentStatus.hasServerError` not getting set to `true` when the `GET enrollment_status` endpoint is not mocked.